### PR TITLE
Removes virologist job

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
As the title says. Removes the job. Whole wing of medical is still there.

Screenshot 2024-04-17 041852

(MSO)
image

## Why It's Good For The Game
It's boring. It's so boring that when it decides to become unboring it just attempts to ruin the fun for everyone else
## Changelog
:cl:
del: Due to a need for company restructuring, virologists have been laid off by CentCom and doctors can optionally take their place.
/:cl:

### CODE REVIEW

**Comment by Higgin** [comment]
@jlsnow301 I am on mobile and haven't the chance to dig in yet - are you leaving the drip?

if so, fuck it, go for it - I already gave docs viro access

**Comment by moocowswag** [comment]
Viro should be reworked to be able to be done cooperatively and even benefit from it so you dont get "too many cooks in the kitchen" when medical doctors have downtime.

**Comment by Bilbo367** [comment]
> Viro should be reworked to be able to be done cooperatively and even benefit from it so you dont get "too many cooks in the kitchen" when medical doctors have downtime.

I think removing the job will make virology a side job (which it should be). This should also give doctors something early on to do besides waiting (which has been why I stopped playing medical). This is good for the same reason scientist covers xeno, toxins, and circuits. Instead of having a specific job for each field.

**Review by Mothblocks — APPROVED** [review]

File: code/game/objects/structures/crates_lockers/closets.dm [hunk 1]
```diff
@@ -125,7 +125,6 @@ GLOBAL_LIST_EMPTY(roundstart_station_closets)
 		"Science" = list("icon_state" = "rd"),
 		"Security" = list("icon_state" = "cap"),
 		"Mining" = list("icon_state" = "mining"),
-		"Virology" = list("icon_state" = "bio_viro"),
```
Comment (Mothblocks) [inline code/game/objects/structures/crates_lockers/closets.dm:128]: Why?

File: code/datums/station_traits/negative_traits.dm [hunk 2]
```diff
@@ -591,10 +591,6 @@
 		//if engineering isnt valid, just send it to the bridge
 		send_supply_pod_to_area(supply_pack_shielding.generate(null), /area/station/command/bridge, /obj/structure/closet/supplypod/centcompod)
 
-	// Let the viro know resistence is futile
-	send_fax_to_area(new /obj/item/paper/fluff/radiation_nebula_virologist(), /area/station/medical/virology, "NT Virology Department", \
-	force = TRUE, force_pod_type = /obj/structure/closet/supplypod/centcompod)
-
```
Comment (Mothblocks) [inline code/datums/station_traits/negative_traits.dm:597]: Why? This sends a fax to the virology room, it's not related to the job

File: code/game/objects/items/toys.dm [hunk 3]
```diff
@@ -1381,10 +1381,6 @@
 	toysay = "I am the law!"
 	toysound = 'sound/runtime/complionator/dredd.ogg'
 
-/obj/item/toy/figure/virologist
-	name = "\improper Virologist action figure"
-	icon_state = "virologist"
-	toysay = "The cure is potassium!"
```
Comment (Mothblocks) [inline code/game/objects/items/toys.dm:1387]: Remove the icon too

File: code/game/objects/structures/crates_lockers/closets.dm (hunk 1 above)
Comment (Mothblocks) [reply code/game/objects/structures/crates_lockers/closets.dm:128]: FMPOV a viro closet carries the world flavor of "the stuff in here is potentially dangerous" and I think that's better than a plain medical closet

File: _maps/map_files/Birdshot/birdshot.dmm [hunk 4]
```diff
@@ -25667,10 +25665,6 @@
 "jKU" = (
 /turf/closed/wall,
 /area/station/engineering/atmos/storage/gas)
-"jLi" = (
-/obj/effect/landmark/start/virologist,
-/turf/open/floor/iron/showroomfloor,
-/area/station/medical/virology)
```
Comment (00-Steven) [inline _maps/map_files/Birdshot/birdshot.dmm:25673]: 1

File: _maps/map_files/Birdshot/birdshot.dmm [hunk 5]
```diff
@@ -35271,13 +35265,6 @@
 /obj/machinery/atmospherics/pipe/smart/simple/dark/hidden,
 /turf/open/floor/iron/grimy,
 /area/station/tcommsat/server)
-"mIa" = (
-/obj/structure/cable,
-/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
-/obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
-/obj/effect/landmark/start/virologist,
-/turf/open/floor/iron/white/small,
-/area/station/medical/virology)
```
Comment (00-Steven) [inline _maps/map_files/Birdshot/birdshot.dmm:35280]: 2

File: _maps/map_files/NorthStar/north_star.dmm [hunk 6]
```diff
@@ -76630,14 +76629,6 @@
...
 /turf/open/floor/pod/dark,
 /area/station/maintenance/floor3/port/aft)
-"tKr" = (
-/obj/machinery/atmospherics/pipe/smart/manifold4w/scrubbers/hidden/layer2,
-/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden/layer4,
-/obj/structure/cable,
-/obj/effect/turf_decal/tile/green/opposingcorners,
-/obj/effect/landmark/start/virologist,
-/turf/open/floor/iron/white,
-/area/station/medical/virology)
```
Comment (00-Steven) [inline _maps/map_files/NorthStar/north_star.dmm:76640]: 3

**Review by 00-Steven — COMMENTED** [review]
It looks like you're currently also removing the area, some atmos, cabling, and flooring on birdshot and northstar?

File: _maps/map_files/Birdshot/birdshot.dmm (hunk 4 above)
Comment (MrMelbert) [reply _maps/map_files/Birdshot/birdshot.dmm:25673]: This is likely just key reshuffling, unless you verified it in map editor nothing to be worried about

**Review by MrMelbert — COMMENTED** [review]
Update jobconfig.toml I think?

**Review by MrMelbert — COMMENTED** [review]
Add +1 total position and +1 spawn position to Medial Doctor

**Comment by GuillaumePrata** [comment]
> I think removing the job will make virology a side job (which it should be). This should also give doctors something early on to do besides waiting

I heavily disagree with this, while I agree doctors could use some extra minor task for downtime, our current system of virology should just be removed as it would not be a good downtime task for doctors.

Bored roundstart doctor goes and make a giga healing disease 10 minutes in, now they actually removed their own content from the round as people don't need medbay as often.

Something something players will optimize the fun out of the game something.

**Comment by jlsnow301** [comment]
> our current system of virology should just be removed as it would not be a good downtime task for doctors.

I'll take what I can get for the moment

**Comment by GuillaumePrata** [comment]
> I'll take what I can get for the moment

Which is smart, hopefully the system for custom diseases go after it, and if the stars align, a rework in a few years.

Unrelated to the PR or comment itself, but I will rant since I am here.

I do wish this was the other way around, removing the custom diseases before the virologist job, being the first defense against biohazards on the station is a cool vibe, and without the custom disease system basically blocking us from smaller changes, we could just let it organically grow with new smaller incremental features for the job over time, or just die eventually if it fails.

Smaller incremental changes is how our well designed jobs got there in the first place, Atmos is my main example of this because it was a nothing burger when I joined, they didn't even had SM or space airlocks access... but Potato, Ghilker, Pickle and many others slowly built something worth a job slot.

Like seriously, virology sucks IMO more because we have been locked on "The department will be reworked soonTM" and literally nothing happens, I've been hearing this for 5 years. If I got an antag token for every time I see people shoting down virology conversations with "It will be reworked soonTM" or "The maintainers only want a full rework" I would get bored of playing antag.

**Comment by RedBaronFlyer** [comment]
I'm kinda eh about this for two reasons:
1. I was kind of hoping that the virology rework in the oven (or at least it was last I checked) was still in the works
2. because I worry that it'll lead to other jobs being merged. I.E. robo and genetics being merged into scientist, coroner being merged into medical doctor, etc.

File: config/jobconfig.toml [hunk 7]
```diff
@@ -183,8 +183,8 @@
 "# Playtime Requirements" = 0
 "# Required Account Age" = 0
 "# Required Character Age" = 0
-"# Spawn Positions" = 3
-"# Total Positions" = 5
+"# Spawn Positions" = 4
+"# Total Positions" = 6
```
Comment (MrMelbert) [inline config/jobconfig.toml:187]: The job datum, `medical_doctor.dm` too. I don't know if it matters but just in case.

**Comment by jlsnow301** [comment]
> 1. I was kind of hoping that the virology rework in the oven (or at least it was last I checked) was still in the works

People have been batting around this as the reason to keep it for a very long time. There are no works in progress to my knowledge. Worse, I don't think there's anything saving it, even with the most inspired of concepts, because the gameplay behind invisible plague is lame for all parties involved

**Comment by Wallemations** [comment]
Damn, can't believe we're getting rid of the green labcoats

Sucks they never had a job to go with them 😔

**Comment by projectkepler-ru** [comment]
I remember distinctly of how much I'd play virologist, do my job and get all the cool virus 25 minutes into the round then just die elsewhere with nothing to do. Also wouldn't hurt eitherway to have doctor fill in virology role since scientist get like 3-4 things to do

**Comment by jlsnow301** [comment]
giphy-2952977743

**Comment by adamkad1** [comment]
Why virologists and not coroners though? People played viro but almost nobody plays coroner. Remove the damn thing and put a scanner on the tray in morgue so we dont need to have a role that is basically obsolete

**Comment by Nabski89** [comment]
Why the virologist toy?

**Comment by MrStonedOne** [comment]
I would suggest everybody who is waiting for some magical viro rewrite to fix the overpowered ness of custom viruses to instead just fucking rebalance custom viruses.

its all numbers on datums, most of them can be tweaked up or down.

and like i said, neutering systems to get their stat buffs without any downsides is a large part of why viro is so op, and should just be removed.

none of these are reasons to throw out a system that used to be very underpowered until coderbus let it get the power creep.

**Comment by CliffracerX** [comment]
Absolutely no reason to remove the virologist toy, and there might be a couple holes left in the maps.  This shouldn't have been speedmerged without even a TM.

**Comment by jlsnow301** [comment]
> Absolutely no reason to remove the virologist toy, and there might be a couple holes left in the maps. This shouldn't have been speedmerged without even a TM.

see above image

**Pull request merged**